src/main.js

```diff
@@ -81,6 +81,14 @@ Vue.directive('rawElement', {
     },
 });
 
+// Register a global custom directive called `v-focus`
+Vue.directive('focus', {
+    // When the bound element is inserted into the DOM...
+    inserted(el) {
+        // Focus the first input element
+        el.getElementsByTagName('input')[0].focus();
```

Comment:
This should check if `el.getElementsByTagName('input')[0]` actually exists before it calls `focus()`